packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -53,6 +55,19 @@ const Duration _kCursorBlinkWaitForStart = Duration(milliseconds: 150);
 // is shown in an obscured text field.
 const int _kObscureShowLatestCharCursorTicks = 3;
 
+/// A map used to disable scrolling shortcuts in text fields.
+///
+/// This is a temporary fix for: https://github.com/flutter/flutter/issues/74191
+final Map<LogicalKeySet, Intent> scrollShortcutOverrides = kIsWeb
```

Comment:
It would be nicer if this could be a private value, rather than a public API. Reducing code duplication probably wins out, but here's another case where I wish Dart had package-private values.

Reply:
It's possible to hide this map by putting it in a new file that's not exported in `widgets.dart`. But my understanding is that's against Flutter's policy of exporting everything.